index.html

```diff
@@ -0,0 +1,62 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <div id="header-img"></div>
+    <div id="header-img-desktop"></div>
+    <div id="padding-ctn">
+        <header id="nav-bar-ctn">
+            <div id="nav-bar">
+                <a>
+                    <p>Portfolio</p>
+                </a>
+                <a>
+                    <p>About</p>
+                </a>
+                <a>
+                    <p>Contact</p>
+                </a>
+            </div>
+        </header>
+        <nav id="toolbar">
+            <div id="menu-button">
+                <object data="icons/icon-menu.svg"></object>
+            </div>
+        </nav>
+        <section id="headline">
+            <h1 id="primary-text">
+                <span id="title-1">Hey there. I'm Gumaro.</span>
+                <span id="title-2">I'm a software developer</span>
+            </h1>
+            <div id="support-text">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you
+                    like what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <div id="social-media">
+                <ul id="soc-med-list">
+                    <li><a href="TBD"><img src="icons/twitter.svg" /></a></li>
+                    <li><a href="TBD"><img src="icons/linkedin.svg" /></a></li>
+                    <li><a href="TBD"><img src="icons/medium.svg" /></a></li>
+                    <li><a href="TBD"><img src="icons/github.svg" /></a></li>
+                    <li><a href="TBD"><img src="icons/angellist.svg" /></a></li>
+                </ul>
+            </div>
+        </section>
+
+        <div id="down-btn-ctn">
+            <button type="button">
+                <img src="icons/union.svg" />
+            </button>
+        </div>
+    </div>
```

Comment:
- [OPTIONAL] I will highly recommend you to apply the structure suggested by the previous reviewer, as your structure has some serious problems regarding to the semantics and SEO.
```
<header>
  <h1>Machine Learning Workshop</h1>
  <nav>
  </nav>
</header>
<main>
  <section id="reg">
    <h2>Machine Learning Workshop Tickets</h2>
  </section>

  <section id="about">
    <h2>What you'll learn</h2>
  </section>

  <section id="teachers">
    <h2>Your Instructors</h2>
    <h3>Hal 9000 <span>&amp;</span> EVE</h3>
  </section>

</main>

<footer>
  <h2>Delivering accessible, performant, standards-compliant websites since 1999.</h2>
</footer>
```